src/components/calcite-tip-manager/calcite-tip-manager.tsx

```diff
@@ -105,11 +106,11 @@ export class CalciteTipManager {
 
   connectedCallback(): void {
     this.setUpTips();
-    this.observer.observe(this.el, { childList: true, subtree: true });
+    this.mutationObserver?.observe(this.el, { childList: true, subtree: true });
```

Comment:
If we want to enforce using `mutationObserver` and `resizeObserver` variables, we can create interfaces for classes to implement as well.